SpectraLogic.SpectraStorageBrokerClient/Model/JobStatusEnum.cs

```diff
@@ -26,24 +25,29 @@ public enum JobStatusEnum
         /// </summary>
         ACTIVE,
 
+        /// <summary>
+        /// The job is canceled
+        /// </summary>
+        CANCELED,
+
         /// <summary>
         /// The job is completed
         /// </summary>
         COMPLETED,
 
         /// <summary>
-        /// The job is paused
+        /// The job has error
         /// </summary>
-        PAUSED,
+        ERROR,
 
         /// <summary>
-        /// The job is canceled
+        /// The job is paused
         /// </summary>
-        CANCELED,
+        PAUSED,
 
         /// <summary>
-        /// The job has error
+        /// The job is retrying
         /// </summary>
-        ERROR
+        RETRY
```

Comment:
RETRY is not a job state other, but an action that results in a new job being created.